parser-typechecker/src/Unison/Runtime/IOSource.hs

```diff
@@ -386,8 +386,8 @@ io.stderr : io.Handle
 io.stderr = Handle "stderr"
 
 -- Throw an I/O error on the left as an effect in `IO`
-io.rethrow : (Either io.Error a) -> {IO} a
-io.rethrow x = match x with
+io.reraise : (Either io.Error a) -> {IO} a
+io.reraise x = match x with
     Either.Left e -> io.IO.throw e
     Either.Right a -> a
```

Comment:
Is this change still relevant to this PR? I think this file could be left alone. If the concern is around name collisions between io and io2, the right thing to do I think is just a `delete.namespace io` at the start of any transcripts expecting to run with the new runtime.

Reply:
Not sure if I'm understanding correctly, but IMO it would be better if old `io` is never added in the first place, when running with the new runtime.

Reply:
Sort of orthogonal.. even if that were done, old IO names could be in the namespace from a prior invocation of ucm. It's not as if you can just delete them then.

Reply:
Sorry, I meant for transcripts that start with an empty namespace and then add stuff to it, just add the correct stuff. 😅

Reply:
Oh gotcha. Yeah that makes sense. I think you need to do `builtin.mergio` to get the stuff in `IOSource` anyway.

@stew I think the point still stands that you could do without this renaming in this PR.

Reply:
we certainly don't need this rename